lib/src/relations/to_many.dart

```diff
@@ -141,8 +143,7 @@ class ToMany<EntityT> extends Object with ListMixin<EntityT> {
       _items.toList(growable: growable);
 
   /// True if there are any changes not yet saved in DB.
-  bool get _hasPendingDbChanges =>
-      _counts.values.where((c) => c != 0).isNotEmpty;
+  bool get _hasPendingDbChanges => _counts.values.any((c) => c != 0);
 
   // bool get _hasPendingDbChanges => _added.isNotEmpty || _removed.isNotEmpty;
```

Comment:
Just noticed this leftover comment, can this be removed?

```suggestion
```

Reply:
removed